fieldservice/models/service_request.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+# from odoo import api, _
+
+
+class ServiceLocation(models.Model):
+    _name = 'service.location'
+    _description = 'Location of the service'
```

Comment:
Is this to indicate a precise location inside a fsm.location?
Should it be linked to a fsm_location_id?